### PR TITLE
Adding text_similarity_reranker retriever specification

### PR DESCRIPTION
**Pull request by pmpailis**
Adding missing `text_similarity_reranker` retriever specification

### CODE REVIEW

File: specification/_types/Retriever.ts [hunk 1]
```diff
@@ -78,3 +80,16 @@ export class RRFRetriever extends RetrieverBase {
...
+  /** The nested retriever which will produce the first-level results, that will later be used for reranking. */
+  retriever: RetrieverContainer
+  /** This value determines how many documents we will consider from the nested retriever.  */
+  rank_window_size?: integer
+  /** Unique identifier of the inference endpoint created using the inference API. */
+  inference_id?: string
+  /** The text snippet used as the basis for similarity comparison */
+  inference_text?: string
+  /** The document field to be used for text similarity comparisons. This field should contain the text that will be evaluated against the inference_text */
+  field?: string
```
Comment (Anaethelion) [inline specification/_types/Retriever.ts:94]: I think we're missing `min_score`
```suggestion
  field?: string
  /** Sets a minimum threshold score for including documents in the re-ranked results. Documents with similarity scores below this threshold will be excluded. Note that score calculations vary depending on the model used. */
  min_score?: float
```

**Review by Anaethelion — CHANGES_REQUESTED** [review]
Looks good! I've left a code suggestion as I think a field was missing.

File: specification/_types/Retriever.ts (hunk 1 above)
Comment (pmpailis) [reply specification/_types/Retriever.ts:94]: Good catch @Anaethelion ! `min_score` has actually been made available for all retrievers through `RetrieverBase`, so updating accordingly!

Comment (pmpailis) [reply specification/_types/Retriever.ts:94]: Updated in 54fd49375ac452a0e6d62a19cbc839a7787829b7

**Comment by pmpailis** [comment]
@Anaethelion  updated and added `min_score` to  base class `RetrieverBase`. Run the tests locally and couldn't find something that is related to these changes, so would appreciate another look whenever you have some time :)

**Review by Anaethelion — APPROVED** [review]
LGTM!

**Pull request merged**